core/src/bank_forks.rs

```diff
@@ -14,6 +14,39 @@ use std::path::{Path, PathBuf};
 use std::sync::Arc;
 use std::time::Instant;
 
+#[derive(Clone, Debug, Eq, PartialEq)]
+pub struct SnapshotConfig {
```

Comment:
should this live in SnapshotUtil ?

Reply:
or just a stand-alone `snapshot_config.rs`?   I don't feel to strongly though

Reply:
SnapshotUtil is not introduced in this PR yet xD, we can move it in the main PR.